.github/workflows/argus-docker-build.yaml

```diff
@@ -44,11 +44,21 @@ on:
         required: false
         type: boolean
         default: false
+      arm_runner:
```

Comment:
```suggestion
      arm_runner_labels:
```

Reply:
In case we want to support other labels 🤷🏻

Reply:
done